player/lua/stats.lua

```diff
@@ -685,6 +685,10 @@ local function append_resolution(s, r, prefix, w_prop, h_prop)
             append(s, r["aspect-name"], {prefix="(", suffix=")", nl="", indent=" ",
                                          prefix_sep="", no_prefix_markup=true})
         end
+        if r["crop-w"] and r["crop-w"] > 0 then
+            append(s, format("[x: %d, y: %d, w: %d, h: %d]", r["crop-x"], r["crop-y"], r["crop-w"],
```

Comment:
Very minor bikeshed but this should probably match the command syntax, i.e. w,h,x,y

Reply:
Here I copied how lavfi-crop is printed and I actually prefer this order. For command it is reversed for convenience when offset is not included.

Reply:
I think it's OK as is.